docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,392 @@
+  annotations:
+    kyma-project.com/timeStamp: <current-time>
+```
+
+### Step 5: Add a parser
+
+Typically, you want to have your logs shipped in a structured format already, so that a backend like OpenSearch can immediately index the content according to the log attributes.
+By default, a LogPipeline will try to parse all logs as JSON document and enrich the record with the parsed attributes on the root record. Thus, logging in JSON format in the application results in structured log records.
+Sometimes, logging in JSON is not an option (the log configuration is not under your control) and the logs are in an unstructured or plain format. To adjust this, you can define your custom [parser](https://docs.fluentbit.io/manual/concepts/data-pipeline/parser) and either activate it with a filter or a Pod annotation.
+
+The following example defines a parser named `dummy_test` using a dedicated `LogParser` resource type:
+
+```yaml
+kind: LogParser
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: dummy_test
+spec:
+  parser:
+    content: |
+      Format regex
+      Regex ^(?<INT>[^ ]+) (?<FLOAT>[^ ]+) (?<BOOL>[^ ]+) (?<STRING>.+)$
+```
+
+The parser is referenced by name in a filter of the pipeline and is activated for all logs of the pipeline.
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  filters:
+    - custom: |
+        Name parser
+        Parser dummy_test
+  outputs:
+    ...
+```
+
+Instead of defining a filter, you can [annotate](https://docs.fluentbit.io/manual/pipeline/filters/kubernetes#kubernetes-annotations) your workload in the following way. Here, the parser is activated only for the annotated workload.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dummy
+  annotations:
+    fluentbit.io/parser: dummy_test
+spec:
+```
+
+## LogPipeline.spec attribute
+
+For details, see the [LogPipeline specification file](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/api/v1alpha1/logpipeline_types.go).
+
+| Parameter | Type | Description |
+|-|-|-|
+| filters | []object | A list of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence as defined. They are executed before logs are buffered, and with that, will not be executed on retries.|
+| filters[].custom | string | The actual filter definition in the syntax of Fluent Bit.|
+| output | object | A [Fluent Bit outputs](https://docs.fluentbit.io/manual/pipeline/outputs).|
+| output.custom | string | The actual output definition in the syntax of Fluent Bit.|
+| variables | | []object | A list of mappings from Kubernetes secret keys to environment variables. Mapped keys are mounted as environment variables, so that they are available as [Variables](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables) in the sections.|
```

Comment:
```suggestion
| variables | | []object | A list of mappings from Kubernetes Secret keys to environment variables. Mapped keys are mounted as environment variables, so that they are available as [Variables](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables) in the sections.|
```
Is "Secret" related to Kubernetes resources must be [capitalized](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms).